boards/common/gd32v/include/cfg_i2c_default.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Gunar Schorcht <gunar@schorcht.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_gd32v
+ * @{
+ *
+ * @file
+ * @brief       Default I2C configuration for GD32VF103 boards
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef CFG_I2C_DEFAULT_H
+#define CFG_I2C_DEFAULT_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    I2C configuration
```

Comment:
```suggestion
 * @brief    I2C configuration
```
should make doxygen happy

Reply:
Yes, but this is a group definition by intention. It makes probably more sense to include it in `generic_exclude_pattern`.

Reply:
This is the way we use for all boards.